client/my-sites/domains/domain-management/email-setup/style.scss

```diff
@@ -16,12 +19,43 @@
 
 .email-setup.foldable-card {
 	.foldable-card__header {
-		padding: 24px;
+		padding: 16px;
+
+		.foldable-card__main {
+			justify-content: space-between;
+			max-width: 100%;
+			margin-right: 0;
+		}
+
+		.foldable-card__main > span > span {
```

Comment:
Is there any reason for the header description not having have a classname? I would prefer to add a CSS rule for the class instead of the selector.

Reply:
That's a good point! Didn't think about that when I was writing this 😅 updated this with a specific class instead of the nested selector 👍